biz.aQute.bnd.maven/src/aQute/bnd/maven/lib/configuration/FileTree.java

```diff
@@ -0,0 +1,111 @@
+package aQute.bnd.maven.lib.configuration;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class FileTree {
+
+    private class _FileTree extends aQute.lib.io.FileTree {
```

Comment:
This should be a static class to avoid passing `this`.

Reply:
Or just changed `aQute.lib.io.FileTree.addFile` from protected to public to avoid the needs to subclass.